src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -194,4 +201,7 @@
   <data name="UnknownToolName" xml:space="preserve">
     <value>Unknown</value>
   </data>
+  <data name="VersionPropertyNotFound_DialogTitle" xml:space="preserve">
+    <value>The sarifLog.version property was not found at the beginning of the sarifLog object. Per the SARIF specification, the version property should appear first.</value>
```

Comment:
>The [](start = 11, length = 3)

"The SARIF specification recommends that the version property appear at the beginning of the log file. This log file does not conform to that recommendation."